bbdo/host.proto

```diff
@@ -31,7 +31,7 @@ message Host {
     STICKY = 2;
   }
```

Comment:
is it better to have this common enum defined outside Host and Service and used by noth?